app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: :PostId
+  has_many :likes, foreign_key: :PostId
+  belongs_to :author, class_name: 'User', foreign_key: :AuthorId
```

Comment:
- [ **OPTIONAL** ] Good job :+1:  to use the correct associations for your the User modal. But as I mentioned in the previous comment, I think it would be better if you use the camel case notation and lowercase on the `AuthorId` and `PostId` kew words (i.e. to be `author_id` and `post_id` ) every where you use it in your App. This for rails good practices.
**I really suggest you apply this approach in all modals and every where you use those notations(e.g.: `migrations`, `schema`,...)**